fms/testing/_internal/hf/model_test_suite.py

```diff
@@ -140,15 +140,17 @@ def test_hf_model_compile_no_graph_breaks(self, fms_hf_model):
         try:
             compiled_model = torch.compile(fms_hf_model, fullgraph=True)
             fms_hf_signature_params = HFModelSignatureParams(
-                compiled_model, self._get_hf_signature_params
+                compiled_model,
+                self._get_hf_signature_params,
+                other_params={"return_dict": True, "attn_algorithm": "math"},
             )
             get_signature(
                 model=fms_hf_signature_params.model,
                 params=fms_hf_signature_params.params,
                 optional_params=fms_hf_signature_params.other_params,
                 logits_getter_fn=fms_hf_signature_params.logits_getter_fn,
             )
-        except TorchDynamoException as e:
+        except TorchDynamoException:
             pytest.fail(f"Failed to get signature of full-graph compiled model")
```

Comment:
shouldn't we output `e` if this happens?

Reply:
Ya, I will add it